src/utils/txma-header.ts

```diff
@@ -0,0 +1,5 @@
+import { Request } from "express";
+
+export function getTxmaHeader(req: Request): string {
+  return (req.headers["txma-audit-encoded"] as string) ?? "";
```

Comment:
If the header isn't present rather than returning an empty string, which could also indicate other issues such as a coding error, would it be better to use a special case value?  If it is missing is it something we'd want to instrument and alert on to flag as a problem?

Reply:
I think it'd be sensible to send a special case value but that needs to be the same value across the programme and I've not seen any mention of doing so in the docs for this work. I've asked in the [initiative Slack channel](https://gds.slack.com/archives/C06GXH2TNSW/p1713255873299489) but we may not get a consensus quickly. 

You're definitely right about monitoring - we should at least have a `WARN` level message if it's not present. I'll add that now.

Reply:
I got a reply 

>  the preference will be not to send the property and any value if you don't have one for it.  This would be consistent with what the cloudfront function being used to generate the audit header does it if can't find a value.  This will help consumers to not need to content check the field as to whether the value is genuine or the missing placeholder text.

We'd want to handle that behaviour at the point we generate the audit events so do you think an empty string is ok here as a 'falsy' value, or would it be better to return `undefined`?

Reply:
Sounds good.  If we don't have a special value then I feel like undefined would be a slightly better option.